packages/default-theme/helpers/pages.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Global routes' map
+ */
+const pagesMap = {
+  'checkout': 'checkout',
+  'account': 'account',
+  'login': 'login',
+  'success-page': 'success-page'
+}
+
+/**
+ * Get the route's path for given page name
+ * @param {string} pageName
+ * @returns {string|null}
+ */
+export function getPagePath(pageName) {
```

Comment:
I agree that we need to maintain the pagesMap, but it's only one place. It would be harder to maintain all the $router.push occurrences if we had it as it was before.